packages/amplify-e2e-tests/src/__tests__/auth_1.test.ts

```diff
@@ -61,4 +71,103 @@ describe('amplify add auth...', () => {
     await addAuthWithDefault(projRoot, {});
     await amplifyPushAuth(projRoot);
   });
+
+  it('...should init a project and add auth with defaultSocial', async () => {
```

Comment:
What is the difference in these tests compared to the ones in `auth_2.test.ts` ?